pkg/cidata/cidata.go

```diff
@@ -413,6 +414,24 @@ func getBootCmds(p []limayaml.Provision) []BootCmds {
 	return bootCmds
 }
 
+// getTimezone returns the timezone of host, to be used in the VM.
+func getTimezone() string {
+	_, GMTOffset := time.Now().Zone()
+	if GMTOffset%3600 != 0 {
+		logrus.Warnf("Host timezone offset is not a multiple of 3600 seconds: %d, skip timezone propagation", GMTOffset)
```

Comment:
The warning is not correct; you are just rounding the timezone offset to full hours, but it is still being applied.

Reply:
Log updated in [97c113c](https://github.com/lima-vm/lima/pull/1931/commits/97c113cde70a6db042accad70143046000a44da2).